src/db/gc/mod.rs

```diff
@@ -95,13 +96,15 @@ const SETTINGS_KEY: &str = "LAST_GC_RUN";
 pub struct MarkAndSweep<DB> {
     db: Arc<DB>,
     get_heaviest_tipset: Box<dyn Fn() -> Arc<Tipset> + Send>,
-    marked: HashSet<u32>,
+    marked: HashSet<Cid>,
```

Comment:
Let's use `SmallCid` to save memory usage

Reply:
Once this has been tested - most definitely!